CMakeLists.txt

```diff
@@ -106,9 +106,9 @@ endif()
 
 get_property(GENERATOR_IS_MULTI_CONFIG GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
 if(GENERATOR_IS_MULTI_CONFIG)
-  set(RegsGen2_RELATIVE_PATH Release/regsgen2${CMAKE_EXECUTABLE_SUFFIX})
+  set(RegsGen2_RELATIVE_PATH Release/regsgen2${CMAKE_HOST_EXECUTABLE_SUFFIX})
 else()
-  set(RegsGen2_RELATIVE_PATH regsgen2${CMAKE_EXECUTABLE_SUFFIX})
+  set(RegsGen2_RELATIVE_PATH regsgen2${CMAKE_HOST_EXECUTABLE_SUFFIX})
```

Comment:
This should be part of the change for the host exe suffix.

Reply:
Doh, bad split. I should have caught that. Will fix